packages/core/admin/admin/src/render.ts

```diff
@@ -4,8 +4,16 @@ import { createRoot } from 'react-dom/client';
 
 import { StrapiApp, StrapiAppConstructorArgs } from './StrapiApp';
 
+interface FeatureConfig {
+  [key: string]: unknown;
```

Comment:
Are the values we receive from configs always strings? Or could they be more?

Reply:
I think the keys are always strings in config, right? 🤔